Prelude/UUnit.lean

```diff
@@ -0,0 +1,10 @@
+
+import Prelude.TypeRep
```

Comment:
I think that when building a prelude it is convenient to have most of the definitions in the same file. Otherwise we might run quickly into mutual dependencies between different files.
You don't need to revert this change, I just wanted to provide some context about my original decision